doc/GUIDE.md

```diff
@@ -608,6 +608,66 @@ public class MyPoint extends Point { }
 Through static inheritance, all the methods in the `Point` class are available through the
 `MyPoint` class as well.
 
+### Array Types
+
+Jextract treats variables (global variables or struct/union fields) with an array type
+specially. For instance, if we have a header file with the following declaration:
+
+```c
+// mylib.h
+
+int FOO_ARRAY[3][5];
```

Comment:
Looking over the entire document, this is the only case where we use all caps. The other section on global variable uses lower case, so probably better to stick with that?